plugins/wallabag.koplugin/main.lua

```diff
@@ -1096,172 +1390,234 @@ Restart KOReader after editing the config file.]]), BD.dirpath(DataStorage:getSe
                     id = "close",
                     callback = function()
                         UIManager:close(self.settings_dialog)
-                    end
+                    end,
                 },
                 {
                     text = _("Info"),
                     callback = function()
                         UIManager:show(InfoMessage:new{ text = text_info })
-                    end
+                    end,
                 },
                 {
                     text = _("Apply"),
                     callback = function()
-                        local myfields = self.settings_dialog:getFields()
-                        self.server_url    = myfields[1]:gsub("/*$", "")  -- remove all trailing "/" slashes
+                        local myfields     = self.settings_dialog:getFields()
```

Comment:
This type of autoformatting diff disaster makes it easy to miss issues like the line right below ftr.